### PR TITLE
adding setup file

### PR DESCRIPTION
**Pull request by ashishsingh18**
adding setup file

### CODE REVIEW

File: DeepMRSeg/data_io.py [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
 _sys.path.append( _os.path.dirname( _sys.argv[0] ) )
```
Comment (jammy270) [inline DeepMRSeg/data_io.py:14]: Comment this line

File: DeepMRSeg/deepmrseg_test.py [hunk 2]
```diff
@@ -16,11 +16,12 @@
 
 import tensorflow as _tf
 import numpy as _np
+import platform as _platform
 
 _sys.path.append( _os.path.dirname( _sys.argv[0] ) )
```
Comment (jammy270) [inline DeepMRSeg/deepmrseg_test.py:21]: Comment this line

File: DeepMRSeg/deepmrseg_test.py [hunk 3]
```diff
@@ -506,6 +508,8 @@ def _main( argv ):
 	_sys.stdout.flush()
 # ENDDEF MAIN
 
+def runFromCMD():
```
Comment (jammy270) [inline DeepMRSeg/deepmrseg_test.py:511]: This function is not required. Call _main directly from setup.py

**Review by jammy270 — CHANGES_REQUESTED** [review]

File: DeepMRSeg/data_io.py (hunk 1 above)
Comment (ashishsingh18) [reply DeepMRSeg/data_io.py]: done

File: DeepMRSeg/deepmrseg_test.py (hunk 2 above)
Comment (ashishsingh18) [reply DeepMRSeg/deepmrseg_test.py]: done

File: DeepMRSeg/deepmrseg_test.py (hunk 3 above)
Comment (ashishsingh18) [reply DeepMRSeg/deepmrseg_test.py]: done. Need to test with data.

**Review by jammy270 — APPROVED** [review]
looks good

**Pull request merged**